src/sparseml/pytorch/image_classification/utils/helpers.py

```diff
@@ -643,3 +684,17 @@ def _download_model_from_zoo_using_recipe(
         Model(recipe_stub),
         recipe_name=recipe_type,
     )
+
+
+def is_image_classification_model(source_path: Union[Path, str]) -> bool:
```

Comment:
Note: Empirically I have discover that not many models in sparsezoo have `arch_key` baked into their checkpoint